src/components/Thought.tsx

```diff
@@ -413,17 +413,14 @@ const ThoughtContainer = ({
 
       <div
         aria-label='thought-container'
+        data-testid={'thought-' + hashPath(path)}
         className={css({
           /* Use line-height to vertically center the text and bullet. We cannot use padding since it messes up the selection. This needs to be overwritten on multiline elements. See ".child .editable" below. */
           /* must match value used in Editable useMultiline */
-          lineHeight: '1.72',
-        })}
-        data-testid={'thought-' + hashPath(path)}
-        style={{
-          // ensure that ThoughtAnnotation is positioned correctly
+          lineHeight: '1.72', // ensure that ThoughtAnnotation is positioned correctly
```

Comment:
```suggestion
          lineHeight: '1.72',
          // ensure that ThoughtAnnotation is positioned correctly
```

I believe this comment applies to the next line (`position: 'relative',`). It's also not a very useful comment, but that's not in the scope of this change.